src/include/sof/math/numbers.h

```diff
@@ -24,6 +24,12 @@
 	typeof(alignment) __alignment = (alignment);	\
 	__size - (__size % __alignment);		\
 })
+
+#define ROUND_UP(size, alignment)				\
+	(((size) % (alignment) == 0)				\
+	? (size)						\
+	: ((size) - ((size) % (alignment)) + (alignment)))
+
```

Comment:
Need to be careful here and reuse the Zephyr maths macros as we've had some naming collisions in the past. @singalsu @kv2019i any reason why we are not using Zephyr C library maths macros today (as I expect we have some duplication).

Reply:
The only difference is that zephyr's ROUND_UP modifies the value each time (I don't know if it's intentional?), while the ROUND_UP I added doesn't modify the value when it is divisible by the given argument. This difference caused me to get incorrect results. So I can modify ROUND_UP in zephyr?